roc.yml

```diff
@@ -0,0 +1,69 @@
+--- 
+GENERATOR: 
+  PackageName: roc 
+  PackageDescription: "Package roc provides Go bindings for libroc"
+  PackageLicense: ""
+  Includes:
+    - ../include/roc/address.h
+    - ../include/roc/config.h
+    - ../include/roc/context.h
+    - ../include/roc/frame.h
+    - ../include/roc/log.h
+    - ../include/roc/platform.h
+    - ../include/roc/receiver.h
+    - ../include/roc/sender.h
+  FlagGroups:
+    - {name: "CFLAGS", flags: [
+        "-I\"../include\"",
```

Comment:
I guess we can remove this block? Since we're removing headers from the repo.

Reply:
I think this block will stay there anyway, we'll just need to pick up the headers from pkg-config somehow? Maybe let's leave it. These bindings will not work with the pkg-config right now whether you have this block (and the cgo comment in the generated files) or not. Maybe let's leave it for future reference?

Reply:
I see. If you want to keep it for future reference, I suggest to replace "../include/roc", which is now a nonexistent path, to "/usr/include/roc", which is where scons installs the headers currently (by default). It would be at least correct path :)

Reply:
will do :)